examples/darwin-framework-tool/commands/common/CHIPCommandBridge.h

```diff
@@ -52,6 +52,9 @@ class CHIPCommandBridge : public Command {
         AddArgument("commissioner-vendor-id", 0, UINT16_MAX, &mCommissionerVendorId,
             "The vendor id to use for darwin-framework-tool. If not provided, chip::VendorId::TestVendor1 (65521, 0xFFF1) will be "
             "used.");
+        AddArgument("use-xpc", 0, 1, &mUseXPC, "This option uses the XPC implementation.");
```

Comment:
Should document what the default behavior is.

Reply:
I have opened https://github.com/project-chip/connectedhomeip/pull/36786 to make it possible to merge `use-xpc` and `use-xpc-service-name` into a single command line argument.

Reply:
How about:

> Use a controller that will connect to an XPC endpoint instead of talking to devices directly

?